Katello/katello
# Fixes #38035 - fix hammer upload existing deb

#### What are the changes introduced in this pull request?
Uploading a deb-package already known to pulp (has already been uploaded to a different repository) results in the package not being available in the publication.

#### What are the testing steps for this pull request?

1. create two deb repositories
2. upload deb-package to repo1 using hammer content-upload
3. upload deb-package to repo2 using hammer content-upload

Package should be available in both repos publication (`/pulp/content/...`).
